test/Templates.Test/Helpers/TemplateTestBase.cs

```diff
@@ -94,6 +107,26 @@ protected void AssertDirectoryExists(string path, bool shouldExist)
             }
         }
 
+        protected void AssertEmptyMigration(string migration)
+        {
+            var fullPath = Path.Combine(TemplateOutputDir, "Data/Migrations");
+            var file = Directory.EnumerateFiles(fullPath).Where(f => f.EndsWith($"{migration}.cs")).FirstOrDefault();
+
+            Assert.NotNull(file);
+            var contents = File.ReadAllText(file);
+
+            var emptyMigration = @"protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
Is this going to work when run on Linux? Will there be new-line issues due to differences in what is in the source file (not generated on Linux) and what is in the generated file?

Reply:
Doesn't look like we need to worry about linux/mac, the top level tests are attributed like so for individual auth:

```
        [ConditionalFact]
        [OSSkipCondition(OperatingSystems.Linux)]
        [OSSkipCondition(OperatingSystems.MacOSX)]```